static/src/stylesheets/module/commercial/glabs/_hosted.scss

```diff
@@ -549,8 +549,8 @@ $hosted-stripe: 28px;
         right: 0;
         bottom: 0;
         position: absolute;
-        background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, .4));
-        border-top: 2px solid transparent;
```

Comment:
gradient should be linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, .4)) (merge error?)